packages/@okta/vuepress-site/docs/release-notes/2021/index.md

```diff
@@ -1,6 +1,39 @@
 ---
 title: Okta API Products Release Notes 2021
 ---
+
+## October
+
+### Monthly Release 2021.10.0
+
+| Change                                                                   | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [AES-GCM encryption support for SAML assertions](#aes-gcm-encryption-support-for-saml-assertions) | October 6, 2021 |
```

Comment:
Suggestion to throw in a link for further info on AES-GCM: https://datatracker.ietf.org/doc/html/rfc5288